edb/lib/schema.edgeql

```diff
@@ -49,6 +49,9 @@ CREATE SCALAR TYPE schema::AccessPolicyAction
 CREATE SCALAR TYPE schema::AccessKind
     EXTENDING enum<`Select`, UpdateRead, UpdateWrite, `Delete`, `Insert`>;
 
+CREATE SCALAR TYPE schema::FuncLanguage
+    EXTENDING enum<EdgeQL, Builtin>;
```

Comment:
I think we should probably _not_ use an enum here, at least not initially, because it might turn out to be too inflexible for when (and if) we allow functions to be written in a language of choice and compiled into common binary format.